src/views/tickets/orders-rsvp.php

```diff
@@ -46,10 +46,15 @@
 				);
 				?>
 			</p>
-			<div class="tribe-tickets attendees-list-optout">
-				<input <?php echo $view->get_restriction_attr( $post_id, esc_attr( $first_attendee['product_id'] ) ); ?> type="checkbox" name="attendee[<?php echo esc_attr( $first_attendee['order_id'] ); ?>][optout]" id="tribe-tickets-attendees-list-optout-<?php echo esc_attr( $first_attendee['order_id'] ); ?>" <?php checked( true, esc_attr( $first_attendee['optout'] ) ) ?>>
-				<label for="tribe-tickets-attendees-list-optout-<?php echo esc_attr( $first_attendee['order_id'] ); ?>"><?php esc_html_e( 'Don\'t list me on the public attendee list', 'event-tickets' ); ?></label>
-			</div>
+			<?php
+				/**
+				* Inject content into the RSVP User Details block on the orders page
+				*
+				* @param array $attendee_group Attendee array
+				* @param WP_Post $post_id Post object that the tickets are tied to
```

Comment:
Looks like the type should be `int` rather than `WP_Post` here - the note after `$post_id` should also be updated or removed to avoid referring to a post object _or_ you could perhaps pass `$post` as the 2nd param instead.